docs/src/main/sphinx/connector/pinot.rst

```diff
@@ -74,6 +74,10 @@ Property name                                             Required   Description
 ``pinot.aggregation-pushdown.enabled``                    No         Push down aggregation queries, default is ``true``.
 ``pinot.count-distinct-pushdown.enabled``                 No         Push down count distinct queries to Pinot, default is ``true``.
 ``pinot.target-segment-page-size``                        No         Max allowed page size for segment query, default is ``1MB``.
+``pinot.extra-http-headers``                              No         Extra headers when sending HTTP based pinot requests to Pinot controller/broker.
+                                                                     Headers are comma-separated, and each header key value pair is colon-separated,
+                                                                     e.g. k1:v1,k2:v2, default is an empty map. Note that controller/broker
+                                                                     authentications will override the ``Authorization`` header if any.
```

Comment:
We should throw an exception instead of override.

Reply:
done